racun-service/README

```diff
@@ -0,0 +1,34 @@
+##Rute
```

Comment:
Generalno nam rute ne trebaju u README-u zato što je cilj da koristimo Swagger ali neka za sada ostane dok ne dodamo Swagger. Takođe, fajl treba da ima .md ekstenziju.